protocol/messenger_communities.go

```diff
@@ -51,6 +51,9 @@ var messageArchiveInterval = 7 * 24 * time.Hour
 // 1 day interval
 var updateActiveMembersInterval = 24 * time.Hour
 
+// 1 hour interval
+var grantUpdateInterval = 1 * time.Hour
```

Comment:
I believe that's too frequent. Once/twice per day should be sufficient, especially since the expiration period is at least 48 hours.